### ISSUE TITLE
Where are the SPDX Matching Guidelines?

### ISSUE THREAD

**Issue by vargenau** [spdx/spdx-3-model#746]
In licenseText, it says:

Users of the licenseText for a License can apply the SPDX Matching Guidelines
when comparing it to another text for matching purposes.

I cannot find the Matching Guidelines in the SPDX 3.0 spec.

**Comment by zvr** [comment]
They are not part of the spec.
They are part of the SPDX License List, at https://spdx.org/licenses/

**Comment by vargenau** [comment]
Hi @zvr 

They were part of the spec. It was Annex B in SPDX 2.3.

And in https://spdx.org/licenses/ the link "[matching guidelines](https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/)" points to SPDX 2.3.

It should be clarified.

**Comment by zvr** [comment]
Yes, in SPDXv2 the License List was part of the spec.
In SPDXv3 it's not.

I agree that it should be stated in the changes between versions 2 and 3.

**Comment by bact** [comment]
The text in the first paragraph of https://spdx.org/licenses/ should also be edited as well.

```text
The SPDX License List is an integral part of the SPDX Specification.
```

**Comment by vargenau** [comment]
Thank you @zvr 

What we should do:

- create a web page containing the matching guidelines independent of SPDX 2.3
- have a link in [licenseText](https://spdx.github.io/spdx-spec/v3.0/model/SimpleLicensing/Properties/licenseText/) to this page
- document in https://spdx.github.io/spdx-spec/v3.0/annexes/diffs-from-previous-editions/ that the license list and the matching guidelines are no longer part of the spec.

**Comment by bact** [comment]
Will be fixed by https://github.com/spdx/spdx-spec/pull/968

**Comment by bact** [comment]
Fixed by https://github.com/spdx/spdx-spec/pull/968 (merged) - can be closed?

**Comment by goneall** [comment]
Closing as fixed by #968

@vargenau - if you disagree or see any additional issues, please open a new issue
